projects/packages/search/tests/php/test-jetpack-search.php

```diff
@@ -1,28 +1,26 @@
-<?php
+<?php // phpcs:ignore WordPress.Files.FileName.InvalidClassFileName: This is necessary to ensure that PHPUnit runs these tests.
 /**
- * Instant Search test cases
+ * Classic Search test cases
  *
  * @package automattic/jetpack
  */
 
-if ( defined( 'IS_WPCOM' ) && IS_WPCOM ) {
-	require_once WPMU_PLUGIN_DIR . '/jetpack-plugin/vendor/autoload_packages.php';
-}
+namespace Automattic\Jetpack\Search;
 
-require_jetpack_file( 'modules/search.php' );
+use PHPUnit\Framework\TestCase;
 
 /**
  * Jetpack_Instant_Search test cases
  */
-class WP_Test_Jetpack_Search extends WP_UnitTestCase {
+class Test_Jetpack_Search extends TestCase {
 
 	/**
 	 * Verify deprecated classes still exist.
 	 *
 	 * @since 10.6.1
 	 */
 	public function test_deprecated_jetpack_search_class() {
-		$search = Jetpack_Search::instance();
+		$search = CLassic_Search::instance();
```

Comment:
Typo. Strange that it doesn't fail when running all the tests. 
It fails when running with a filter tough: `--filter=Test_Jetpack_Search`.

```suggestion
		$search = Classic_Search::instance();
```

Reply:
Ah, strange indeed. Fixed!